src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -214,15 +213,15 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
   const basedFailureRate = shapeToolData?.supertypes?.supertypes?.hasFailureRate?.estimate?.value;
   const requiredFailureRate = shapeToolData?.supertypes?.hasFailureRate?.requirement?.upperBound;
 
-  const FailureRateBox = ({ value, label, rate, selected, classes }) => (
+  const FailureRateBox = ({ value, label, rate, selected }) => (
     <Box display="flex" flexDirection="row" alignItems="center">
       <FormControlLabel
         value={value}
-        control={<Radio className={classes.black} />}
+        control={<Radio className={classes.selected} />}
```

Comment:
Shouldn't it be selectable?

Reply:
changed to: style={{ color: theme.main.black }}